examples/redux-saga-typescript-graphql-fluentui/constants/env.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * Environment variables
+ */
+export const Env = {
+  NODE_ENV: process.env.NODE_ENV,
+  API_GATEWAY_ENTRYPOINT:
+    process.env.API_GATEWAY_ENTRYPOINT || 'http://localhost:9000',
+}
```

Comment:
Let's try to remove this file, it's better to use environment variables directly